src/index.html

```diff
@@ -1,13 +1,309 @@
+          +1&#32;234&#32;5555&#45;55&#45;55
+        </a>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__content-bottom__call-to-order">
+          call to order
+        </a>
+      </div>
+    </aside>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="main">
+      <section class="products" id="products">
+
+        <h2 class="title products__title">Recommended</h2>
+
+        <div class="products__contatiner">
+
+          <article class="product-card">
+            <img
+              class="product-card__image"
+              src="images/sellphone.png"
+              alt="mobile phone"
+            >
+            <p class="name product-card__name">Phone (1)</p>
+
+            <p class="description product-card__description">
+              <span class="description description__fragment">Designed with instinct, to bring joy back to the everyday.</span>
+              Through the Glyph Interface, a perfected OS and exceptional dual camera. All startlingly fast.
+            </p>
+
+            <p class="price">$ 499.00</p>
+          </article>
+
+
+          <article class="product-card">
+            <img
+                class="product-card__image"
+                src="images/headphones-stick.png"
+                alt="headphones"
+              >
+            <p class="name product-card__name">Ear (2)</p>
+
+            <p class="description product-card__description">
+              Perfecting the path from artist to ear. So that music sounds just as it was intended.
+            </p>
+            <p class="price">$ 149.00</p>
+          </article>
+
+          <article class="product-card">
+            <img
+              class="product-card__image"
+              src="images/ear-stick.png"
+              alt="Ear-stick"
+            >
+            <p class="name product-card__name">Ear (stick)</p>
+
+            <p class="description product-card__description">
+              Tech you can&#96;t feel. Leaving just space for sound to come to life. Powered by custom sound technology.
+            </p>
+            <p class="price">$ 119.00</p>
+          </article>
+        </div>
+      </section>
+
+
+
+      <section class="category" id="store">
+
+        <h2 class="title category__title">
+          Browse Nothing
+          <br>
+          products by category
+        </h2>
+
+        <section class="category-block">
+          <a href="#" class="category-block__item--1">
+            <img
+              class="category-block__image"
+              src="images/parrots-phone.png"
+              alt="parrots phone"
+            >
+          </a>
+
+          <a href="#" class="category-block__item--2">
+            <img
+              class="category-block__image"
+              src="images/headphones-hand.png"
+              alt="headphones hand"
+            >
+          </a>
+        </section>
+
+        <h3 class="name category-block__name">All products</h3>
+
+
```

Comment:
```suggestion
```
1 blank line between sibling elements (there should never be more than 1 empty line)